HACKING

```diff
@@ -219,6 +219,10 @@ Coding
   ``gint``, use a ``gchar`` for individual (ASCII/UTF-8) string
   characters rather than ``gint``, and use a ``guint`` for integers
   which cannot be negative rather than ``gint``.
+* Avoid using untyped pointers (e.g. gpointer) where practical.
```

Comment:
Not that it matters, but I think `void` is technically an "incomplete type" rather than "untyped".

Since this rule is basically the same thing as the one above (presumably why it was placed here), I wonder if it would be better to just make it an example/extension on the previous rule?

Reply:
Good point, I've merged it with the above rule.